.all-contributorsrc

```diff
@@ -0,0 +1,27 @@
+{
+  "projectName": "marvin",
+  "projectOwner": "ajtorres9",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": ["readme.md"],
+  "imageSize": 100,
+  "commit": false,
+  "contributorsPerLine": 7,
+  "badgeTemplate": "  <a href=\"#contributors\"><img alt=\"All Contributors\" src=\"https://flat.badgen.net/badge/all%20contributors/<%= contributors.length %>/orange\"></a>",
+  "contributors": [
+    {
+      "login": "ajtorres9",
+      "name": "Andrew Torres",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/450495?v=4",
+      "profile": "https://andrewjtorr.es",
+      "contributions": ["code", "doc", "infra"]
+    },
+    {
+      "login": "brian-dlee",
+      "name": "Brian Lee",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/15238587?v=4",
+      "profile": "https://github.com/brian-dlee",
+      "contributions": ["doc"]
```

Comment:
Damn! Just doc! 🤣

Reply:
I'll add more when I have the bot add you. 😉